dask/dataframe/methods.py

```diff
@@ -446,13 +446,14 @@ def concat_pandas(
     join="outer",
     uniform=False,
     filter_warning=True,
-    ignore_order=False,
     ignore_index=False,
     **kwargs
 ):
     if axis == 1:
         return pd.concat(dfs, axis=axis, join=join, **kwargs)
 
+    ignore_order = kwargs.get("ignore_order", False)
```

Comment:
Would also suggest using `pop` here as `**kwargs` is getting forwarded to Pandas, which doesn't know how to handle this (causing CI failures atm)

```suggestion
    ignore_order = kwargs.pop("ignore_order", False)
```